cstring.h

```diff
@@ -609,6 +610,34 @@ TString stringSubstring(TString s, size_t pos, size_t len) {
     return res;
 }
 
+TString stringFormat(const char* format, ...) {
```

Comment:
Please add the function definition somewhere at the beginning next to the rest
And add some [tests](https://github.com/Tnirpps/cstring/blob/main/tests/main.c) for your function